zips/zip-0225.rst

```diff
@@ -145,15 +145,15 @@ Transaction Format
 |``1``                        |``flagsOrchard``          |``byte``                                |An 8-bit value representing a set of flags. Ordered from LSB to MSB: |
 |                             |                          |                                        | * ``enableSpendsOrchard``                                           |
 |                             |                          |                                        | * ``enableOutputsOrchard``                                          |
-|                             |                          |                                        | * The remaining bits are set to ``0``.                              |
+|                             |                          |                                        | * The remaining bits are set to :math:`0\!`.                        |
 +-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
 |``8``                        |``valueBalanceOrchard``   |``int64``                               |The net value of Orchard spends minus outputs.                       |
 +-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
 |``32``                       |``anchorOrchard``         |``byte[32]``                            |A root of the Orchard note commitment tree at some block             |
 |                             |                          |                                        |height in the past.                                                  |
 +-----------------------------+--------------------------+----------------------------------------+---------------------------------------------------------------------+
 |``varies``                   |``sizeProofsOrchard``     |``compactSize``                         |Length in bytes of ``proofsOrchard``. Value is                       |
-|                             |                          |                                        |:math:`2720 + 2272 \cdot \mathtt{nActionsOrchard}`.                  |
+|                             |                          |                                        |:math:`2720 + 2272 \cdot \mathtt{nActionsOrchard}\!`.                |
```

Comment:
Same question.

Reply:
The automatic spacing adjustment when math is followed by a punctuation mark (or preceded by an open paren) is intentionally only applied to `$...$`. If we applied it to `:math:` then it would interfere with using that for column-sensitive markup.